notify/notify.yaml

```diff
@@ -8,7 +8,7 @@ paths:
   /team-webhooks:
     get:
       summary: Get all webhooks
```

Comment:
you want to change this "summary" field as well

Reply:
Fixed.